mobile-app/src/i18n/locales/he.json

```
@@ -132,6 +147,53 @@
+      "serialNumber": "מספר סידורי",
+      "serialNumberPlaceholder": "למשל, LG123456789",
+      "categoryStatus": "קטגוריה וסטטוס",
+      "category": "קטגוריה",
+      "status": "סטטוס",
+      "supplier": "ספק",
+      "supplierPlaceholder": "למשל, Home Depot, Best Buy",
+      "purchasePrice": "מחיר רכישה",
+      "tags": "תגיות",
+      "tagsPlaceholder": "למשל, חסכוני באנרגיה, אחריות, חדש",
+      "validationError": "שגיאת אימות",
+      "requiredFieldsMessage": "אנא מלא את כל השדות הנדרשים",
```

Biome (1.9.4)

[error] The key status was already declared.

This where a duplicated key was declared again.

If a key is defined multiple times, only the last definition takes effect. Previous definitions are ignored.

(lint/suspicious/noDuplicateObjectKeys)